gstcefsrc.cc

```diff
@@ -329,6 +330,25 @@ class App : public CefApp
     if (src->chromium_debug_port >= 0) {
       command_line->AppendSwitchWithValue("remote-debugging-port", g_strdup_printf ("%i", src->chromium_debug_port));
     }
```

Comment:
Wouldn't say the same for the rest of the flags, but this could be removed with this change

Reply:
I'm sorry, I should have reacted earlier, but removing the property is an API break, I'd rather have it stay and simply update the property documentation to mark it as `deprecated: use chrome-extra-flags instead` :)